Dockerfile

```diff
@@ -21,7 +21,10 @@ ENV     RUSTFLAGS="-C target-feature=-crt-static"
 # Create dummy main.rs files for each workspace member to be able to compile all the dependencies
 RUN     find . -type d -name "meilisearch-*" | xargs -I{} sh -c 'mkdir {}/src; echo "fn main() { }" > {}/src/main.rs;'
 # Use `cargo build` instead of `cargo vendor` because we need to not only download but compile dependencies too
-RUN     $HOME/.cargo/bin/cargo build --release
+RUN     if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+            export JEMALLOC_SYS_WITH_LG_PAGE=16; \
+        fi && \
+        $HOME/.cargo/bin/cargo build --release
 # Cleanup dummy main.rs files
 RUN     find . -path "*/src/main.rs" -delete
 
```

Comment:
Hi @curquiza, I tested it successfully on arm64 machine. But compiling the image is just too slow 😂.
BTW, can we remove the `cargo build` above? It does not seem necessary.

Reply:
Hum, I would rather remove this line in another PR to ensure we do not any mistake by introducing JEMALLOC_SYS_LG_PAGE